cmd/cycloid/stacks/get-config.go

```diff
@@ -71,13 +71,19 @@ func getConfig(cmd *cobra.Command, args []string) error {
 	}
 
 	ref, err := cmd.Flags().GetString("ref")
+	if err != nil {
+		return errors.Wrap(err, "unable to get ref flag")
+	}
 	if len(args) >= 1 && ref == "" {
 		ref = args[0]
 	} else if ref == "" {
 		return fmt.Errorf("missing ref argument.")
 	}
```

Comment:
Here I am checking if the first arg (that could be the ref) is set, if not, I check if the flag returned a value.

All the error case is handled here.